components/groups.js

```diff
@@ -666,34 +662,34 @@ SteamCommunity.prototype.getGroupJoinRequests = function(gid, callback) {
  * @param {function} callback - Takes only an Error object/null as the first argument
  */
 SteamCommunity.prototype.respondToGroupJoinRequests = function(gid, steamIDs, approve, callback) {
-	if (typeof gid === 'string') {
+	if (typeof gid == 'string') {
 		gid = new SteamID(gid);
 	}
-	
-	var rgAccounts = (!Array.isArray(steamIDs) ? [steamIDs] : steamIDs).map(sid => sid.toString());
+
+	let rgAccounts = (!Array.isArray(steamIDs) ? [steamIDs] : steamIDs).map(sid => sid.toString());
 
 	this.httpRequestPost({
-		"uri": "https://steamcommunity.com/gid/" + gid.getSteamID64() + "/joinRequestsManage",
-		"form": {
-			"rgAccounts": rgAccounts,
-			"bapprove": approve ? "1" : "0",
-			"json": "1",
-			"sessionID": this.getSessionID()
+		uri: `https://steamcommunity.com/gid/${gid.getSteamID64()}/joinRequestsManage`,
+		form: {
+			rgAccounts: rgAccounts,
+			bapprove: approve ? '1' : '0',
+			json: '1',
```

Comment:
Any reason to use strings instead of numbers here?

Reply:
It reinforces understanding that application/x-www-form-urlencoded requests are sent as strings and have no concept of other data types.